lib/timeline/component/graph2d_types/points.js

```diff
@@ -31,8 +31,69 @@ Points.prototype.draw = function(dataset, group, framework, offset) {
  */
 Points.draw = function (dataset, group, framework, offset) {
   if (offset === undefined) {offset = 0;}
+  
+  var callback = undefined;
+  // check for the graph2d onRender
+  if (framework.options.drawPoints.onRender) {
+	  allback = framework.options.drawPoints.onRender;
+  }
+  
+  // override it with the group onRender if defined
+  if (group.options.drawPoints.onRender) {
+  	callback = group.options.drawPoints.onRender;
+  }
+
+  if (callback) {
```

Comment:
maybe check if it is a function? The validator is not used on the groups at the moment.